ui/app/js/controllers/permission_scheduled_changes_controller.js

```diff
@@ -112,6 +134,72 @@ function($scope, $routeParams, $location, $timeout, Permissions, Rules, Search,
     });
   };
 
+  $scope.signoff = function(sc) {
+    var modalInstance = $modal.open({
+      templateUrl: "signoff_modal.html",
+      controller: "SignoffCtrl",
+      backdrop: "static",
+      resolve: {
```

Comment:
I'll give this a try, too. It might be a bit weird since they take different args, though.